context/test_notebooks.py

```diff
@@ -31,6 +31,8 @@
     # context on these being skipped: https://github.com/rapidsai/cuspatial/pull/1407
     'cuspatial/cuspatial_api_examples.ipynb',
     'cuspatial/nyc_taxi_years_correlation.ipynb',
+    # details: https://github.com/rapidsai/docker/issues/726
+    'cuspatial/trajectory_clustering.ipynb',
```

Comment:
Proposing that we skip this notebook as well, to get CI working here.

The work of getting this working is described in #726.